text/0013-iaas.md

```diff
@@ -0,0 +1,58 @@
+
+# Guide-level explanation
+On blockchain networks, a user is represented as a public key (or as any derived identifier from that, e.g. [Ethereum Address](https://ethereum.stackexchange.com/questions/3542/how-are-ethereum-addresses-generated)) of an [asymmetric keypair](https://en.wikipedia.org/wiki/Public-key_cryptography).
+
+A user can be represented on a P2P storage network as well if the storage network can derive data address from any particular Public Key derived identifier.
+
+Let's take for example Ethereum Swarm: it has content addressing where the data itself defines its address and there is also single owner chunk, where the address is calculated by hashing an Ethereum address and an arbitrary topic together.
+The single-owner chunks are only accepted by signing the BMT hash of their payload with the corresponding key related to the Ethereum Address used in address calculation. 
+On this storage network, the single-owner chunks locate the Identity Anchored Address Space (IAAS): the public key of the asymmetric keypair determines the address space where the writing is only possible with possessing the private key, and the user's public key derived identifier (Ethereum address) is required to read from it.
+
+Because on blockchain networks the user account creation equals to generating asymmetric keypair and there is at least one storage network that handles IAAS, The Fair Data Society user account creation also takes only creating an asymmetric keypair.
+
+# Reference-level explanation
+The user creation has been already implemented in [fdp-storage](https://github.com/fairDataSociety/fdp-storage) JavaScript library.
+
+FDP account creation:
+```js
+import { FdpStorage } from '@fairdatasociety/fdp-storage'
+
+const fdp = new FdpStorage(BEE_API_URL, BEE_DEBUG_API_URL)
+const user = fdp.account.createWallet()
+```
+
+The `BEE_API_URL` and `BEE_DEBUG_API_URL` are bee network configuration data for its APIs (e.g. 'http://localhost:1633', 'http://localhost:1635').
+Later it will be changed after Swarm introduces token based authentication on its endpoints.
+
+The wallet creation uses the [createRandom](https://docs.ethers.io/v5/api/signer/#Wallet-createRandom) function of the `ethers` library.
+
+This wallet can be used for Pods, folders creations and file uploads of the user.
+
+# Drawbacks
+TBD
+
+# Rationale and alternatives
+This approach employs widely-used concepts for wallet handling and assymetric keypair generation.
+
+The proposal aimed at setting the least amount of requirements for creating a Fair Data Society user for using FDS services.
+
+# Prior art
+- Book of Swarm v1.0 pre-release 7 - 2.2.3 Single-owner chunks 
+
+# Future possibilities
+The wallet handling is a crutial topic in this concern, but the login workflow will be elaborated in a different FIP.
+
```

Comment:
add option for subdomain registrations sub.username.fds ?

Reply:
that would be more like the USNS (User Storage NameSpace). That can be elaborated in the mentioned FIP.